### DISCUSSION TITLE
Disabling add to home screen prompt

### DISCUSSION THREAD

**Discussion by NoahJelen** [bromite/bromite#2342] [Q&A] [answered]
Is there a way to disable this annoying prompt? I have disabled every single PWA related flag in chrome://flags with no luck at all.
![Screenshot_20220920-115536](https://user-images.githubusercontent.com/29718341/191343331-a657401d-b219-4f2c-b71a-eb658a8b4bbe.png)

**Comment by csagan5** [comment, marked as answer]
I will disable it on next release; it can currently be disabled by disabling NTP tiles, which can only be disabled by disabling history.
